test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1

```diff
@@ -465,4 +465,172 @@ er
             $ps.HadErrors | Should -BeFalse
             $output.Replace("`r","").Replace(" ",".").Replace("`n","``") | Should -BeExactly $expectedTable.Replace("`r","").Replace(" ",".").Replace("`n","``")
         }
-}
+
+        It "Format-Table should correctly render rows: <variation>" -TestCases @(
+            @{ view = "Default"; widths = 4,7,5; variation = "narrow values"; values = [PSCustomObject]@{First=1;Second=2;Third=3}; wrap = $false; expectedTable = @"
+
+Long*Header2*Heade
+Long*********r3
+Head
+er
+----*-------*-----
+1**********2***3
+
+
+
+"@ },
+@{ view = "Default"; widths = 4,7,5; variation = "narrow values with wrap"; values = [PSCustomObject]@{First=1;Second=2;Third=3}; wrap = $true; expectedTable = @"
```

Comment:
Seems like inconsistent indenting of the test cases.  My preference is to have each new test case align with line 470.

Reply:
Fixed.